episodes/quality-control-and-manual-segmentation.md

```diff
@@ -0,0 +1,547 @@
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- What is segmentation?
+- How do we manually segment images in Napari?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Explain some common quality control steps e.g. assessing an image's histogram, 
+checking clipping/dynamic range
+
+- Explain what a segmentation, mask and label image are
+
+- Create a label layer in Napari and use some of its manual segmentation tools 
+(e.g. paintbrush/fill bucket)
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+For the next few episodes, we will work through an example of counting the 
+number of cells in an image. 
+
+First, let's open one of Napari's sample images with:  
+`File > Open Sample > napari builtins > Cells (3D + 2Ch)`
+
+![](fig/cells-napari.png){alt="A screenshot of a flourescence microscopy image 
+of some cells in Napari"}
+
+## Quality control
+
+The first step in any image processing workflow is always quality control. We 
+have to check that the images we acquired at the microscope capture the features 
+we are interested in, with a reasonable signal to noise ratio (recall we 
+discussed signal to noise ratio in the [last episode
+](choosing-acquisition-settings.md#signal-to-noise-ratio)).
+
+In order to count our cells, we will need to see individual nuclei in our 
+images. As there is one nucleus per cell, we will use the nucleus number as the 
+equivalent of cell number. If we look at our cells image in Napari, we can 
+clearly see individual nuclei highlighted in green. If we zoom in we can see 
+some noise (as a slight 'graininess' over the image), but this doesn't interfere 
+with being able to see the locations or sizes of all nuclei.
+
+Next, it's good practice to check the image histogram, as we covered in the 
+[choosing acquisition settings episode
+](choosing-acquisition-settings.md#initial-quality-control). We'll do this in 
+the exercise below:
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Histogram quality control
+
+Plot the image histogram with `napari matplotlib`:
+
+- Do you see any issues with the histogram?
+
+If you need a refresher on how to use `napari matplotlib`, check out the 
+[image display episode](image-display.md#napari-plugins).
+
+:::::::::::::::::::::::: solution 
+
+### Solution
+
+In the top menu bar of Napari select:  
+`Plugins > napari Matplotlib > Histogram`
+
+Make sure you have the 'nuclei' layer selected in the layer list (should be 
+highlighted in blue).
+
+![](fig/nuclei-histogram.png){alt="A histogram of the 29th z slice of Napari's 
+cell sample image"}
+
+Note that as this image is 3D `napari matplotlib` only shows the histogram of 
+the current z slice (in this case z=29 as shown at the top of the histogram). 
+Moving the slider at the bottom of the viewer, will update the histogram to show 
+different slices.
+
+If we look at the brightest part of the image, near z=29, we can see that there 
```

Comment:
What do the white vertical lines on the histograms mean? If they are max and minimum values why is the minimum not zero? Or was this all covered in another episode.

Reply:
I'll clarify this in the text. The white lines show the current contrast limits, which by default aren't set to the full range from min to max. It's a known issue that is in the process of being fixed on Napari - https://github.com/napari/napari/issues/6534#issuecomment-1853644167

Reply:
I think the added text makes this clear now.

Reply:
Agree!